src/Ilios/CliBundle/Command/SetRootUserCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Ilios\CliBundle\Command;
+
+use Ilios\CoreBundle\Entity\Manager\UserManager;
+use Ilios\CoreBundle\Entity\UserInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Grants root-level privileges to a given user.
+ *
+ * Class SetRootUserCommand
+ * @package Ilios\CliBundle\Command
+ */
+class SetRootUserCommand extends Command
+{
+    /**
+     * @var string
+     */
+    const COMMAND_NAME = 'ilios:maintenance:set-root-user';
```

Comment:
set-root-user implies to me that there is only one.  should probably be called add-root-user.